src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.tsx

```diff
@@ -101,7 +101,8 @@ function ImageRenderer({tnode}: ImageRendererProps) {
                                     return;
                                 }
 
-                                const route = ROUTES.ATTACHMENTS?.getRoute(reportID ?? '-1', type, source, accountID, isAttachmentOrReceipt);
+                                const imageHrefLink = tnode.parent?.attributes?.href;
```

Comment:
The variable name imageLink is already meaningful enough, so we don’t need to use imageHrefLink
```suggestion
                                const imageLink = tnode.parent?.attributes?.href;
```